### PR TITLE
bug-fix: Fix carousel swipe flickering issue + Add missing default font-family to template root

### PR DESCRIPTION
**Pull request by liamcho**
### Changelogs
- Fixed a bug where swiping carousel in mobile view displaying flickering effect
- Added missing default font-family value to template root

### CODE REVIEW

File: src/ui/Carousel/index.tsx [hunk 1]
```diff
@@ -63,119 +79,208 @@ export function Carousel({
...
+    }
   };
 
   const handleTouchEnd = () => {
-    if (dragging !== null) {
-      setDragging(null);
-    }
-    onDragEnd();
+    handleDragEndForMobile();
+  };
```
Comment (HoonBaek) [inline src/ui/Carousel/index.tsx:184]: same?
```
const handleTouchEnd = handleDragEndForMobile;
```

File: src/ui/Carousel/index.tsx [hunk 2]
```diff
@@ -63,119 +79,208 @@ export function Carousel({
...
+
+    if (newOffset === draggingInfo.offset) return;
+    if (draggingInfo.dragging) {
+      setDraggingInfo({
+        ...draggingInfo,
+        offset: newOffset,
+      });
+      return;
+    }
+    if (deltaY > deltaX + 15) {
```
Comment (HoonBaek) [inline src/ui/Carousel/index.tsx:167]: better to make these `15` to constance

**Review by HoonBaek — APPROVED** [review]
LGTM

File: src/ui/Carousel/index.tsx (hunk 1 above)
Comment (liamcho) [reply src/ui/Carousel/index.tsx:184]: yep its same

**Pull request merged**